src/lib/logical_query_plan/lqp_utils.cpp

```diff
@@ -134,12 +134,12 @@ void lqp_replace_node(const std::shared_ptr<AbstractLQPNode>& original_node,
   /**
    * Untie this node from the LQP
    */
-  original_node->set_left_input(nullptr);
-  original_node->set_right_input(nullptr);
+  original_node->set_input_left(nullptr);
+  original_node->set_input_right(nullptr);
 }
 
-void lqp_remove_node(const std::shared_ptr<AbstractLQPNode>& node, const AllowRightInput allow_right_input) {
-  Assert(allow_right_input == AllowRightInput::Yes || !node->right_input(),
+void lqp_remove_node(const std::shared_ptr<AbstractLQPNode>& node, const AllowInputRight allow_input_right) {
+  Assert(allow_input_right == AllowInputRight::Yes || !node->input_right(),
```

Comment:
This sounds somewhat yodaesque. I think I'd leave it the way it was before.

Reply:
+1 for `yodaesque`. :D